tests/unit/test_file_ds.py

```diff
@@ -4,4 +4,63 @@
 Author: Eugeniu Costetchi
 Email: costezki.eugen@gmail.com 
 """
+import pathlib
+import tempfile
 
+import pytest
+
+from eds.adapters.file_ds import FileDataSource
+
+BASE_DIR = tempfile.TemporaryDirectory()
```

Comment:
you don't seem to be using it.

if you need a temporary file or folder for the tests use the [`tmpdir`](https://docs.pytest.org/en/stable/tmpdir.html#the-tmpdir-fixture) fixture.

Reply:
removed